app/app/route.js

```diff
@@ -15,8 +15,12 @@ export default Ember.Route.extend({
     return tokens.join(' - ');
   },
 
+  setupController: function(controller, model){
+    controller.set('model', model);
+    controller.set('stack', model.get('stack.content'));
```

Comment:
> model.get('stack.content`)

@bantic  Getting stack.content still seems to be required here in order to make sure `permittable` works.  Am I missing a better way to handle this?